achievements/contribubit.achievement.js

```diff
@@ -0,0 +1,33 @@
+var _ = require('lodash');
+var console = process.console;
+
+var TheGodfatherConsigliere = {
+	name: 'TheGodfatherConsigliere',
```

Comment:
This name is what the user sees. It should have spaces

the `var XXX = {}` name of the entire achievement shouldn't have spaces (since it's a javascript var)